portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,23 +26,30 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private final List<String> myComments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<String> comments = new ArrayList<>();
-    comments.add("Comment 1: This blog is interesting!");
-    comments.add("Comment 2: Well known information. Not too interesting.");
-    comments.add("Comment 3: Needs some more development, but cool concept!");
-    String commentJSON = convertToJson(comments);
+    String commentJSON = convertToJson(myComments);
```

Comment:
You can use a synchronized or a concurrent list implementation for your Servlet fields to avoid any synchronization problems. Like  `java.util.concurrent.ConcurrentLinkedQueue` or `Collections.synchronizedList(new ArrayList<>())`. I know you're moving away from this array implementation into a DataStore model. So this comment is just an FYI for future.

Reply:
ooh ok! I'll look into that thank you!